src/scss/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,54 @@
+/**
+ * Breadcrumbs Component
+ * =================================
+ * Used for website navigation
+ *
+ */
+
+$breadcrumb-font-weight : $font-weight-bold !default;
+
+//pixel size of the breadcrumb chevron
+$breadcrumb-chevron-height : 6px;
+$breadcrumb-chevron-width : 10px;
+$breadcrumb-chevron-point-right : -90deg;
+
+.c-breadcrumb {
+    width: 100%;
+    @include font-size(mid, false);
+    line-height: 3;
+}
+
+.c-breadcrumb-item {
+    float: left;
+    color: $grey--dark;
+
+    &--home {
```

Comment:
Prefer to include the full selector rather than using `&` shorthands, just as it's easier to search or find and replace should someone need to find selectors later.